test/extended/testdata/builds/subscription-content/buildconfig-subscription-content-rhel8.yaml

```
@@ -32,7 +32,7 @@ spec:
     dockerfile: |
       FROM registry.access.redhat.com/ubi8/ubi:latest
       RUN rm -rf /etc/rhsm-host
-      RUN yum --enablerepo=codeready-builder-for-rhel-8-x86_64-rpms install \
+      RUN yum --enablerepo=codeready-builder-for-rhel-8-$arch-rpms install \
```

Use output from `arch` command:

```suggestion
      RUN yum --enablerepo=codeready-builder-for-rhel-8-$(arch)-rpms install \
```

---

Thanks Adam, Of course! that makes a lot of sense. super simple. I'll mention it to @chandra0007